memtable/write_buffer_manager.cc

```diff
@@ -177,16 +178,17 @@ void WriteBufferManager::MaybeFlushLocked(DB* this_db) {
   if (!ShouldFlush()) {
     return;
   }
+  constexpr size_t kCandidateSize = 2;
```

Comment:
Why we need 2 candidates?

Reply:
It's for L239. We need at least one candidate to force flush without considering whether it will be destroyed or not. Once it is flushed, other normal tablets can replace it and be flushed later.